### PR TITLE
PS 6086 - Release Notes for 8.0.18-9

### PR DESCRIPTION
**Pull request by patrickbirch**
Merged 5.6 and 5.7 release notes
updated the release notes for 8.0 
No "New Features" in this release

### CODE REVIEW

**Comment by dutow** [comment]
"Merged 5.6 and 5.7 release notes" - I don't see a merge commit? Was this a cherry pick?

**Comment by patrickbirch** [comment]
There are three commits.
I merged the 5.6 release notes from my remote branch into 5.7 and then the 5.7 release notes from my remote branch into 8.0.

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+- Setting the ``none`` value for :variable:`slow_query_log_use_global_control`
+  generates an error. Bugs fixed :psbug:`5813`.
+
+- If `pam_krb5
+  <https://docs.oracle.com/cd/E88353_01/html/E37853/pam-krb5-7.html>`__ allows the
+  user to change their password, and the password expired, a new password may
+  cause a server crash. Bug fixed :psbug:`6023`.
+
+- A check if any temporary tables should be logged to binlog would cause
+  failures. Bug fixed :psbug:`5181`.
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-8.0.18-9.rst]: "would cause failures" is probably better stated "may cause the server to crash"

**Review by george-lorch — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.18-9.rst]: Changed.

**Review by george-lorch — APPROVED** [review]

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+:psbug:`6065`,
+:psbug:`6078`,
+:psbug:`6057`,
+:psbug:`6062`,
+:psbug:`6111`,
+:psbug:`6063`,
+:psbug:`6111`,
+:psbug:`6117`,
+:psbug:`6073`, and
+:psbug:`6080`.
```
Comment (dutow) [inline doc/source/release-notes/Percona-Server-8.0.18-9.rst]: Also PS-6123.

Also: shouldn't this list be ordered?

**Review by dutow — COMMENTED** [review]
Looks good other than one more bug and an optional comment.

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+- Setting the ``none`` value for :variable:`slow_query_log_use_global_control`
+  generates an error. Bugs fixed :psbug:`5813`.
+
+- If `pam_krb5
+  <https://docs.oracle.com/cd/E88353_01/html/E37853/pam-krb5-7.html>`__ allows the
+  user to change their password, and the password expired, a new password may
+  cause a server crash. Bug fixed :psbug:`6023`.
+
+- A check if any temporary tables should be logged to binlog may cause the
```
Comment (kamil-holubicki) [inline doc/source/release-notes/Percona-Server-8.0.18-9.rst]: This happened only in debug build (incorrect assertion was triggered)

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  sqlexception. Bug fixed :psbug:`6094`. (Upstream :mysqlbug:`97682`)
+
+- When executing ``SHOW GLOBAL STATUS``, the variables may return incorrect
+  values. Bug fixed :psbug:`5966`.
+
+- If the page tracker is enabled, and the innodb start fails, the page tracker
+  thread does not exit at shutdown. Bug fixed :psbug:`5642`.
+
+- The memory storage engine detected an incorrect ``full`` condition even 
+  though the space contained deleted files and space could be reused.
```
Comment (kamil-holubicki) [inline doc/source/release-notes/Percona-Server-8.0.18-9.rst]: I would rather say: "space contained reusable memory chunks released by deleted records'

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.18-9.rst]: Added in 6123.
Checked with Zsolt on ordering the Other Bugs list.

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  Bug fixed :psbug:`1469`
+
+Other bugs fixed:
+
+:psbug:`6051`,
+:psbug:`5876`,
+:psbug:`5964`,
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-8.0.18-9.rst]: This is not a public bug - just an internal CI improvement

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+:psbug:`5996`,
+:psbug:`6021`,
+:psbug:`6122`,
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-8.0.18-9.rst]: An internal bug - wasn't even visible to users as was reported and fixed during 8.0.18 merge

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+:psbug:`6026`,
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-8.0.18-9.rst]: Not a bug - internal "Null merge" task

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+:psbug:`6028`,
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-8.0.18-9.rst]: Likewise, internal CI improvement

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+:psbug:`6052`,
+:psbug:`4775`,
+:psbug:`5836`,
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-8.0.18-9.rst]: This is an upstream bug that was fixed in 8.0.18 (https://bugs.mysql.com/bug.php?id=96449)

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+:psbug:`6123`,
+:psbug:`5819`,
+:psbug:`5836`,
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-8.0.18-9.rst:55]: Duplicate, 56, 36 is already in the list

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+:psbug:`6054`,
+:psbug:`6056`,
+:psbug:`6058`,
+:psbug:`6059`,
+:psbug:`6065`,
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-8.0.18-9.rst]: This patch was reverted.

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.18-9.rst]: updated.

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst (hunk 4 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.18-9.rst]: updated.

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+:psbug:`6078`,
+:psbug:`6057`,
+:psbug:`6062`,
+:psbug:`6111`,
+:psbug:`6063`,
+:psbug:`6111`,
+:psbug:`6117`,
+:psbug:`6073`, and
+:psbug:`6080`.
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-8.0.18-9.rst]: Internal CI-only improvement

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+:psbug:`5819`,
+:psbug:`5836`,
+:psbug:`6054`,
+:psbug:`6056`,
+:psbug:`6058`,
+:psbug:`6059`,
+:psbug:`6065`,
+:psbug:`6078`,
+:psbug:`6057`,
+:psbug:`6062`,
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-8.0.18-9.rst]: Internal CI-only task

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst [hunk 14]
```diff
@@ -0,0 +1,76 @@
...
+:psbug:`6111`,
+:psbug:`6063`,
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-8.0.18-9.rst]: Internal CI-only task

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst [hunk 15]
```diff
@@ -0,0 +1,76 @@
...
+:psbug:`6111`,
+:psbug:`6117`,
```
Comment (percona-ysorokin) [inline doc/source/release-notes/Percona-Server-8.0.18-9.rst]: Internal "Release tasks" ticket

**Review by percona-ysorokin — CHANGES_REQUESTED** [review]

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst (hunk 5 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.18-9.rst]: removed.

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst (hunk 6 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.18-9.rst]: removed

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst (hunk 7 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.18-9.rst]: removed

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst (hunk 8 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.18-9.rst]: removed

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst (hunk 9 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.18-9.rst]: removed

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst (hunk 10 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.18-9.rst:55]: removed

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst (hunk 11 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.18-9.rst]: removed

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst (hunk 12 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.18-9.rst]: removed

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst (hunk 13 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.18-9.rst]: removed

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst (hunk 14 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.18-9.rst]: removed

File: doc/source/release-notes/Percona-Server-8.0.18-9.rst (hunk 15 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.18-9.rst]: removed

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Review by kamil-holubicki — APPROVED** [review]

**Pull request merged**